components/dashboard/src/workspaces/ConnectToSSHModal.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useState } from "react";
+import Modal from "../components/Modal";
+import Tooltip from "../components/Tooltip";
+import copy from "../images/copy.svg";
+import CodeText from "../components/CodeText";
+
+function InputWithCopy(props: { value: string; tip?: string; className?: string }) {
+    const [copied, setCopied] = useState<boolean>(false);
+    const copyToClipboard = (text: string) => {
+        const el = document.createElement("textarea");
+        el.value = text;
+        document.body.appendChild(el);
+        el.select();
+        try {
+            document.execCommand("copy");
+        } finally {
+            document.body.removeChild(el);
+        }
+        setCopied(true);
+        setTimeout(() => setCopied(false), 2000);
+    };
+    const tip = props.tip ?? "Click to copy";
+    return (
+        <div className={`w-full relative ${props.className ?? ""}`}>
+            <input
+                disabled={true}
+                readOnly={true}
+                autoFocus
+                className="w-full pr-8 overscroll-none"
+                type="text"
+                defaultValue={props.value}
+            />
+            <div className="cursor-pointer" onClick={() => copyToClipboard(props.value)}>
+                <div className="absolute top-1/3 right-3">
+                    <Tooltip content={copied ? "Copied!" : tip}>
```

Comment:
**suggestion**: Minor rephrase! 😁 

```suggestion
                    <Tooltip content={copied ? "Copied" : tip}>
```

We should probably also align other instances of this or create a reusable component in future iterations. However, feel free to leave this as is!

https://github.com/gitpod-io/gitpod/blob/00e08df666edff200edf760ba26f40803c5ebfde/components/dashboard/src/teams/Members.tsx#L296

https://github.com/gitpod-io/gitpod/blob/2f0b9462b0451f0974434fa36bd757420ef04e34/components/dashboard/src/settings/Teams.tsx#L682